playbooks/hosts/amc_production/group_vars/edx

```diff
@@ -0,0 +1,204 @@
+---
+
+EDXAPP_MONGO_HOSTS: ['prod-amc-mongo-1', 'prod-amc-mongo-2', 'prod-amc-mongo-3']
+EDXAPP_MONGO_USER: 'edxapp'
+EDXAPP_MONGO_PASSWORD: "{{ secret_EDXAPP_MONGO_PASSWORD }}"
+EDXAPP_MONGO_DB_NAME: 'edxapp'
+EDXAPP_MYSQL_HOST: '{{ _mysql_host }}'
+
+
+EDXAPP_LMS_ENV: 'lms.envs.amc'
+EDXAPP_CMS_ENV: 'cms.envs.amc'
+
+EDXAPP_AUTH_EXTRA:
+  APPSEMBLER_SECRET_KEY: "{{secret_APPSEMBLER_SECRET_KEY}}"
+  APPSEMBLER_AMC_API_BASE: "https://amc.appsembler.com/api"
+  MAILGUN_ACCESS_KEY: "{{secret_MAILGUN_ACCESS_KEY}}"
+  MAILGUN_SERVER_NAME: "{{secret_MAILGUN_SERVER_NAME}}"
+
+edx_platform_repo: "https://github.com/appsembler/edx-platform.git"
+edx_ansible_source_repo: "https://github.com/appsembler/configuration"
+
+EDX_VERSION: named-release/dogwood
+edx_platform_version: "appsembler/amc/develop"
+configuration_version: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+certs_version: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+forum_version: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+xqueue_version: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+INSIGHTS_VERSION: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+
+EDXAPP_FEATURES:
+  AUTH_USE_OPENID_PROVIDER: true  # default: true
+  ENABLE_OAUTH2_PROVIDER: true  # default: false
+  ALLOW_ALL_ADVANCED_COMPONENTS: true  # default: false
+  ENABLE_SYSADMIN_DASHBOARD: True  # default: false
+  #ENABLE_CREATOR_GROUP: true  # default: false
+  CERTIFICATES_ENABLED: true  # default: true
+  ENABLE_DISCUSSION_SERVICE: true  # default: true
+  ENABLE_INSTRUCTOR_ANALYTICS: false  # default: false
```

Comment:
It's probably safer to have the analytics service running on a separate VM, so that it can be updated independently from the Open edX app server(s).  Although I know you can also selectively run Ansible roles on a VM.